content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -100,12 +101,12 @@ metadata:
   # "namespace" omitted since ClusterRoles are not namespaced
   name: secret-reader
 rules:
-- apiGroups: [""]
-  #
-  # at the HTTP level, the name of the resource for accessing Secret
-  # objects is "secrets"
-  resources: ["secrets"]
-  verbs: ["get", "watch", "list"]
+  - apiGroups: [""]
+    #
+    # at the HTTP level, the name of the resource for accessing Secret
+    # objects is "secrets"
+    resources: ["secrets"]
```

Comment:
How about keeping the original manifests verbatim but convert them to `{{< codenew >}}` shortcodes?

Reply:
Good call, @sftim - I've removed these markdown changes to keep this relevant to the PR title and will create another PR with the markdown and shortcode edits.